### ISSUE TITLE
[Feature Request] Make the links of a post take less space

### ISSUE THREAD

**Issue by Palloxin** [nullxception/boorusphere#49]
![IMG_20221212_145055](https://user-images.githubusercontent.com/75091899/207062306-e083b474-33dd-406c-95e8-153f6589f0f5.jpg)
The space dedicated to links is too much. I would like the links to be compact (nearer) OR something to contain them like this:

https://user-images.githubusercontent.com/75091899/207063710-4cced0bc-8ec9-4b26-a6f5-93b2155ec565.mp4

Personally the only link that might be useful is the source link since sometime i want to track the artist from the source. the other links that come from the actual server (safebooru etc..) can be contained in there. If you have questions, ask

**Comment by nullxception** [comment]
From UI perspective, it's nice, but that's horrible UX.
Details Page is literally for "detail information", if you need to expand things first to reveal another detail, then it defeat the purpose.

**Comment by Map987** [comment]
Yes, I am trying to collect source links to find and access artworks.
